README.md

```diff
@@ -21,6 +21,7 @@ The syntax of poorly closed `p` and `img` elements is cleaned up.
 Allowing particular urls as a `src` to an iframe tag by filtering hostnames is also supported.
 
 HTML comments are not preserved.
+Additionally, `sanitize-html` escapes _ALL_ text content - this means that ampersands, quotes, etc are converted to their equivalent HTML character references (`&` --> `&amp;`).
```

Comment:
Not true for quotes, except within attributes where of course it is necessary.

Reply:
Ah, my bad, I'll fix that in a second! Is there somewhere I can check for all the characters that are escaped? I'm assuming it's ampersands and greater-than / less-than signs chiefly?